.github/workflows/release.yml

```
@@ -0,0 +1,22 @@
+on:
+  push:
+    branches:
+      - main
```

issue: the main branch of this repo is `master`

```suggestion
      - master
```

---

Sorry I'm a bit new to this -- does marking as resolved apply your suggested change automatically or will I need to update my branch?

---

@ficcdaf just push changes to the same branch you used to create this PR. Resolving the comment does not do anything except hide the comment.